dashboard/db/schema.rb

```diff
@@ -1176,9 +1179,6 @@
     t.index ["program_type"], name: "index_workshops_on_program_type", using: :btree
   end
 
-  add_foreign_key "authored_hint_view_requests", "levels"
-  add_foreign_key "authored_hint_view_requests", "scripts"
-  add_foreign_key "authored_hint_view_requests", "users"
```

Comment:
These removals look unintentional/unrelated- maybe something out of sync in your local DB?

Reply:
My local DB was out of sync.